### PR TITLE
Initial setup and header & footer implementation

### PR DESCRIPTION
**Pull request by MinboyKim**
### 📎 Title
Initial setup and header & footer implementation

### 📃 Changes

* Fonts configuration
* metadata(favicon, title, description)
* CSS Reset
---
* Header
* Footer
* PR Template

#### Fonts configuration
1. Configure to apply font file using Next.js `localFont`
2. I thought there were only Pretendard fonts, but there are a few more. There may be a few more because there is no design system. 😭 If I missed anything please add it
3. By default, pretendard is applied to the body
4. We can apply GoogleSans fonts as font-google and SUITE fonts as font-suite className.

#### metadata(favicon, title, description)
1. I didn't have a proper favicon image, so I made one using a picture in Figma, but it looks a bit fat. It looks like we need to get proper image and change it.

#### CSS Reset
1. We should have done this before, but we forgot. I did it roughly. If you need more, please add it.

#### Header & Footer
1. Implemented according to the figma documentation.

#### PR Template
1. I forgot to apply the PR template. Added 🫠

### 📌 Important
Since CI/CD is linked to the main branch, it must be merged into the main branch to see the results. I think it would have been okay even if there was no develop branch in the middle due to the small number of developers. 😓
### 🎇 Screenshots
스크린샷 2024-07-02 오전 6 19 47

### 💫 ETC
Let’s work hard together 🫡

### CODE REVIEW

**Comment by clicelee** [comment]
I appreciate you setting up the initial configuration. 
Nice work!

File: app/_component/Footer.tsx
Comment (clicelee) [inline app/_component/Footer.tsx:1]: I want to ask why the component folder starts with _ under score

**Review by clicelee — APPROVED** [review]
I appreciate you setting up the initial configuration.
Nice work!

File: app/_component/Footer.tsx
Comment (MinboyKim) [reply app/_component/Footer.tsx:1]: This is to specify that the folder does not need to be routed. In fact, if page.tsx does not exist, routing will not occur, but I did it to display it more explicitly. For more details click [here.](https://nextjs.org/docs/app/building-your-application/routing/colocation) They are called `private folders.`

**Pull request merged**